RealmSwift/Aliases.swift

```diff
@@ -74,6 +74,38 @@ extension ObjectBase {
      transactions it will be called at some point in the future after the write
      transaction is committed.
 
+     If no key paths are given, the block will be executed on any insertion,
+     modification, or deletion for all object properties and nested linked
+     properties (up to a depth of 4). If a key path or key paths are provided,
+     then the block will be called for changes which occur on those key paths,
+     or links to those key paths (up to a depth of four). For example, if:
```

Comment:
A filter can exceed the depth of 4 restriction.

Reply:
Resolved in [b8e9ee0](https://github.com/realm/realm-cocoa/pull/7249/commits/b8e9ee02afed35f06f77bd688886ad6b7aea627d)